interface/forms/newpatient/common.php

```diff
@@ -183,7 +183,13 @@ class="css_button link_submit" target='Main' onClick="top.restoreSession()">
  </div>
 
 <br> <br>
-
+<?php
+if (!$viewmode) {
+    $query = "SELECT pc_aid FROM openemr_postcalendar_events WHERE pc_eventDate = ? and pc_pid =?";
```

Comment:
Also aren't we supporting recurring appointments? I don't think this will work in that case.